### ISSUE TITLE
Adding a NAVBAR for various purposes for the webpage.

### ISSUE THREAD

**Issue by aashimgarg** [publiclab/community-toolbox#319]
Add the navbar at the top of the page.
Make it responsive.
Add various things in the navbar.
Make it sticky.

**Comment by aashimgarg** [comment]
Hey @Rishabh570 and @jywarren do you think i should work on this issue and you guys wants this on your webpage.

**Comment by aashimgarg** [comment]
Please suggest.

**Comment by Suiiyush** [comment]
If no one is currently working o this issue can i work on it, I plant to add a fully responsive nav bar with logo. It will be sticky too

**Comment by YogeshSharma01** [comment]
Hello there anyone working on this ?

**Comment by jywarren** [comment]
We'd love help with it if you'd like to -- how might this look, can we try
to brainstorm a sketch or mockup? Thanks! What if we started minimally like
the navbar in https://unterbahn.com ?

**Comment by YogeshSharma01** [comment]
@jywarren yeah we can start with this minimally like the navbar in https://unterbahn.com

**Comment by YogeshSharma01** [comment]
And I would like to work on this just let me know that what all navigation sections should be added in the navbar,  like about, contact etc.

**Comment by jywarren** [comment]
Ok, awesome - well, what are the different headers that go down the page?
We can add an anchor to each one. I wonder if we can also have some be
navbar items but others just fold away under a "more" option? We are using
Bootstrap so there are some different nav design options:

https://getbootstrap.com/docs/4.4/components/

Thank you! What do you think?

**Comment by YogeshSharma01** [comment]
Hi, @jywarren according to you I come up with a sketch where all the important sections of the page are in the header. So, just click on which section a user want to go and all other important links I have folded into more section ... but as what I think is it is more of learning page especially for first-timers so here in this second sketch I have mention all the links and important sections of the page in the header so any first-timer can easily find all the important things related to his/her need. And extra links I've added in more button of the header, and we can also add the logo of PublicLab on the very left corner of the Header.
![Header](https://user-images.githubusercontent.com/51679200/103336621-c4aea480-4a9e-11eb-8eb9-7921f4ad8849.png)
Sketch-1 
![Header-2](https://user-images.githubusercontent.com/51679200/103336651-d98b3800-4a9e-11eb-940d-c01ac76342db.png)
Sketch-2

**Comment by YogeshSharma01** [comment]
Hi @jywarren What do you think about which one I should start work on?

**Comment by YogeshSharma01** [comment]
I give a pull request, please review @jywarren

**Comment by jywarren** [comment]
Hi my apologies, can you link me to the pull request? I've been very busy and so sorry to be slow replying here.

**Comment by jywarren** [comment]
I guess I think we shouldn't have too many top-level menu items because its a bit overwhelming. So, I like the second mockup!

**Comment by YogeshSharma01** [comment]
Hi, @jywarren link https://github.com/publiclab/community-toolbox/pull/420

**Comment by YogeshSharma01** [comment]
Hi, @jywarren have you review my changes?
